fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexFunctionHelper.java

```diff
@@ -264,4 +285,11 @@ public static IndexAggregateFunction countUpdates(@Nonnull KeyExpression by) {
         return indexMaintainerForAggregateFunction(store, function, recordTypeNames)
                 .map(i -> function.cloneWithIndex(i.state.index.getName()));
     }
+
+    public static Optional<IndexAggregateFunction> bindAggregateFunctionCall(@Nonnull FDBRecordStore store,
```

Comment:
Javadoc? Does this return the function that can matched by an index?

Reply:
Yes. Will add a javadoc.